src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/ContentFieldIndexProvider.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json.Linq;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.Data;
+using YesSql.Indexes;
+
+namespace OrchardCore.ContentFields.Indexing
+{
+    // Remark: 
+
+    public class ContentFieldIndex : MapIndex
+    {
+        public string ContentType { get; set; }
+        public string ContentPart { get; set; }
+        public string ContentField { get; set; }
+    }
+
+    public class ContentFieldIndexProvider : IndexProvider<ContentItem>, IScopedIndexProvider
+    {
+        private readonly IServiceProvider _serviceProvider;
+        private readonly HashSet<string> _ignoredTypes = new HashSet<string>();
+        private IContentDefinitionManager _contentDefinitionManager;
+
+        public ContentFieldIndexProvider(IServiceProvider serviceProvider)
+        {
+            _serviceProvider = serviceProvider;
+        }
+
+        public override void Describe(DescribeContext<ContentItem> context)
+        {
+            context.For<ContentFieldIndex>()
```

Comment:
I still need to override that method but this could be all removed unless we want to have a ContentFieldIndex table added in the migration.